pa4.sql

```diff
@@ -0,0 +1,249 @@
+('Thriving in the Digital Age: Strategies for Success', '978-5432109876', 1, 2021, 29.99, 3, 4),
+('The Art of Mindfulness: Living in the Present', '978-1357924680', 3, 2019, 12.99, 4, 5),
+('Pride and Prejudice', '9780141439518', 1, 2000, 9.99, 1, 1), 
+('Great Expectations', '9780141439563', 1, 2001, 12.99, 2, 1), 
+('The Old Man and the Sea', '9780684801223', 1, 2002, 10.99, 3, 1), 
+('Murder on the Orient Express', '9780062693662', 1, 2002, 11.99, 4, 3), 
+('The Fellowship of the Ring', '9780547928210', 1, 2004, 15.99, 5, 5), 
+('Beloved', '9781400033416', 1, 2005, 14.99, 6, 1), 
+('One Hundred Years of Solitude', '9780060883287', 1, 2006, 13.99, 7, 2);
+
+INSERT INTO customers (name, address, phone, email, password_code) VALUES
+('Alice Johnson', '123 Maple St, Smalltown', '555-123-4567', 'alice.johnson@gmail.com', 'SecurePass123!'),
+('Bob Smith', '456 Oak St, Villagetown', '555-987-6543', 'bob.smith@yahoo.com', 'StrongPass456$'),
+('Charlie Brown', '789 Pine St, Hamletville', '555-111-2222', 'charlie.brown@hotmail.com', 'SafePassword789!'),
+('David Davis', '654 Elm St, Countryside', '555-333-4444', 'david.davis@example.com', 'ProtectedPassABC$'),
+('Emma Wilson', '987 Cedar St, Cityville', '555-555-5555', 'emma.wilson@outlook.com', 'SecretPassDEF!'),
+('Emma Johnson', '789 Elm Road, Village, Country', '+1122334455', 'emma@example.com', 'new_hashed_password_1'),
+('Oliver Brown', '101 Pine Lane, City, Country', '+1122334455', 'oliver@example.com', 'new_hashed_password_2'),
+('Ava Wilson', '202 Maple Street, Town, Country', '+9988776655', 'ava@example.com', 'new_hashed_password_3'),
+('Liam Anderson', '303 Cedar Avenue, Village, Country', '+5544332211', 'liam@example.com', 'new_hashed_password_4'),
+('Charlotte Taylor', '404 Birch Road, City, Country', '+7711223344', 'charlotte@example.com', 'new_hashed_password_5'),
+('Noah Smith', '123 Main Street, City, Country', '+1234567890', 'noah@example.com', 'new_hashed_password_6'),
+('Isabella Doe', '456 Oak Avenue, Town, Country', '+1987654321', 'isabella@example.com', 'new_hashed_password_7');
+
+INSERT INTO loans (customer_id, book_id, loan_date, due_date, return_date) VALUES
+(1, 1, '2024-01-01', '2024-01-15', '2024-01-14'),
+(2, 2, '2024-01-05', '2024-01-20', NULL),
+(3, 3, '2024-01-10', '2024-01-25', NULL),
+(4, 4, '2024-01-15', '2024-01-30', NULL),
+(1, 5, '2024-01-20', '2024-02-04', NULL),
+(6, 6, '2024-02-01', '2024-02-15', NULL),
+(7, 7, '2024-02-02', '2024-02-16', NULL),
+(1, 8, '2024-02-03', '2024-02-17', NULL),
+(9, 9, '2024-02-04', '2024-02-18', NULL),
+(10, 10, '2024-02-05', '2024-02-19', NULL),
+(1, 11, '2024-02-06', '2024-02-20', NULL),
+(12, 12, '2024-02-07', '2024-02-21', NULL);
+
+INSERT INTO genres_mtm (book_id, genre_id) 
+VALUES 
+(1, 1), (1, 2),
+(2, 2),
+(3, 3),
+(4, 4),
+(5, 5),
+(6, 1),
+(7, 1),
+(8, 1),
+(9, 2),
+(10, 5),
+(11, 1),
+(12, 2);
+
+INSERT INTO customers_mtm (customer_id, loan_id) 
```

Comment:
Why do we have again and again schema creation scripts and data population? They should be present just once in the scope of 1st or 2nd P/A.